components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterEditor.scala

```diff
@@ -73,35 +70,27 @@ object PeriodParameterEditor {
   - add https://github.com/jmrozanec/cron-utils to model classpath
   - add CronDefinitionBuilder, CronParser and CronType to additional classes in ExpressionConfig
  */
-case object CronParameterEditor extends SimpleParameterEditor
+case object CronParameterEditor extends ParameterEditor with StaticParameterEditor
 
 @JsonCodec case class FixedValuesParameterEditor(possibleValues: List[FixedExpressionValue])
-    extends SimpleParameterEditor
+    extends ParameterEditor
+    with StaticParameterEditor
 
 @JsonCodec case class FixedValuesWithIconParameterEditor(possibleValues: List[FixedExpressionValueWithIcon])
-    extends SimpleParameterEditor
+    extends ParameterEditor
+    with StaticParameterEditor
 
 @JsonCodec case class FixedValuesWithRadioParameterEditor(possibleValues: List[FixedExpressionValue])
-    extends SimpleParameterEditor
+    extends ParameterEditor
+    with StaticParameterEditor
 
 // TODO: currently only supports String/Boolean/Long dictionaries (same set of supported types as AdditionalDataValue)
 @JsonCodec case class DictParameterEditor(
     dictId: String // dictId must be present in ExpressionConfigDefinition.dictionaries
-) extends SimpleParameterEditor
+) extends ParameterEditor
+    with StaticParameterEditor
 
-@JsonCodec case class DualParameterEditor(simpleEditor: SimpleParameterEditor, defaultMode: DualEditorMode)
-    extends ParameterEditor
-
-object DualParameterEditor {
-
-  implicit val dualEditorModeEncoder: Encoder[DualEditorMode] = {
-    new Encoder[DualEditorMode] {
-      override def apply(editorMode: DualEditorMode): Json = Encoder.encodeString(editorMode.name())
-    }
-  }
-
-  implicit val decodeDualEditorMode: Decoder[DualEditorMode] = {
-    Decoder.decodeString.emapTry(name => Try(DualEditorMode.fromName(name)))
-  }
+// Editor with static values (without SpEL)
+@ConfiguredJsonCodec sealed trait StaticParameterEditor
 
-}
+case object ConstStringParameterEditor extends StaticParameterEditor
```

Comment:
`Const` sounds like there is some constant value that can't be changed. Maybe let's call it `StaticStringParameterEditor`? Alos, let's move this object closer to rest of static editors.

Reply:
Done